HW_11-12.md

```diff
@@ -0,0 +1,84 @@
+## Task 1:
+
+As a result of each point, you should provide a corresponding command.
+
+localhost - your CentOS VM running in VirtualBox
+
+remotehost - 18.221.144.175 (public IP)
+
+webserver - 172.31.45.237 (private IP)
+
+#### 1.1. SSH to remotehost using username and password provided to you in Slack. Log out from remotehost.
+```
+ssh Rustam_Mukhomedishin@18.221.144.175
+exit
+```
+
+#### 1.2. Generate new SSH key-pair on your localhost with name "hw-5" (keys should be created in ~/.ssh folder).
+```
+ssh-keygen -t rsa
+path: /home/rustam2/.ssh/hw-6
+```
+
+#### 1.3. Set up key-based authentication, so that you can SSH to  remotehost  without password.
+```
+ssh-copy-id -i ~/.ssh/hw-6 Rustam_Mukhomedishin@18.221.144.175
+```
+
+#### 1.4. SSH to remotehost without password. Log out from remotehost.
+```
+ssh Rustam_Mukhomedishin@18.221.144.175
+exit
+```
+I coldn't login without password. Permissions on files and directories are correct. I couldn't detect a problem yet.
+
+#### 1.5. Create SSH config file, so that you can SSH to remotehost simply running `ssh remotehost` command. As a result, provide output of command `cat ~/.ssh/config`.
+```
+Host remotehost
+  HostName 18.221.144.175
+  User Rustam_Mukhomedishin
+```
+
+#### 1.6. Using command line utility (curl or telnet) verify that there are some webserver running on port 80 of webserver.  Notice that webserver has a private network IP, so you can access it only from the same network (when you are on remotehost that runs in the same private network). Log out from remotehost.
+```
+ssh remotehost
+curl -v http://172.31.45.237:80
+exit
+
+Never Gonna Give You Up :D
+```
+
+#### 1.7. Using SSH setup port forwarding, so that you can reach  webserver from your localhost (choose any free local port you like).
+```
+ssh -L 2022:172.31.45.237:80 remotehost
+```
+I couldn't open the web-page on localhost:2022. Is my command corrected?
```

Comment:
Check your ports with `netstat -tnlp`
Your forwarding is attached exactly to the address that you'll see near the specified port